### ISSUE TITLE
How to install and run on Mac OS Catalina

### ISSUE THREAD

**Issue by LFranco25** [Zarbuz/FileToVox#47]
Good afternoon I have read previous threads whom held the same issue as mine. Unfortuntely, none of the solution listed have helped so far. Every time I download the ".osx" zip file, I do not get a ".exe" file. I just get a file named "FileToVox", which when I try to open it, it just opens in the "text/Edit" app on Mac. Any advice on this matter would truly be appreciated. Thank you.

**Comment by ahx-code** [comment]
You need to make it executable. For instance:

```
chmod +x FileToVox
```

Then you need to specify input and output like:

```
./FileToVox -i input.ply -o input
```

Then you will have an output on your current dir. In my case FileToVox gave me the following warnings:

```
[INFO] FileToVox v1.14.0.0
[INFO] Author: @Zarbuz. Contact : https://twitter.com/Zarbuz
[INFO] Specified input path: dresser_0201.ply
[INFO] Specified output path: dresser.vox
[INFO] Specified output path: /Users/ahx/Desktop/file2vox/dresser.vox
[INFO] Start reading PLY data...
[INFO] Done.
[WARNING] No voxels to quantize, skipping this part...
[INFO] Vox Width: 0
[INFO] Vox Length: 0
[INFO] Vox Height: 0
[INFO] Total blocks: 0
[INFO] Started to write chunks ...
[INFO] Done.
[INFO] Check total blocks after conversion: 0
[INFO] Done.
```

As a result, I have a blank output.

**Comment by LFranco25** [comment]
Thank you fo 4 the reply @ahx-code and I also apologize for the late response. Just stepping back into the office. I am sorry to be difficult but unfortunately I am extremely new to the world of coding. Through a tutorial on YouTube I found this filetovox app. Unfortunately the user was utilizing pc to demonstrate the apps functionality. 

So far the only thing I did before your response, was to download the .osx version of the app as shown below...
![Screen Shot 2022-01-17 at 10 20 07 AM](https://user-images.githubusercontent.com/97756014/149796257-b49fa440-413c-43b7-bac0-5c7ebe3a5dc3.png)

When I open the folder I have these files within...
![Screen Shot 2022-01-17 at 10 21 31 AM](https://user-images.githubusercontent.com/97756014/149796452-d69578d1-0c2b-434f-b6b2-e4b055b909cd.png)

This is where I get stuck and do not know how to proceed. I open the "FileToVox" file and this is what opens...
![Screen Shot 2022-01-17 at 10 23 28 AM](https://user-images.githubusercontent.com/97756014/149796880-445c5042-28de-4f2f-9be4-6aed9e4a75c7.png)
